src/renderer/components/ft-card/ft-card.css

```
@@ -1,8 +1,7 @@
 .ft-card {
   background-color: var(--card-bg-color);
   margin: 8px;
-	padding: 16px;
-	padding-top: 3px;
-	padding-bottom: 16px;
+	padding-block: 3px 16px;
```

Minor indentation issue
```suggestion
  padding-block: 3px 16px;
```